lib/report.js

```diff
@@ -74,7 +76,8 @@ class Report {
       reports.create(_reporter, {
         skipEmpty: false,
         skipFull: this.skipFull,
-        maxCols: process.stdout.columns || 100
+        maxCols: process.stdout.columns || 100,
+        projectRoot: this.projectRoot
```

Comment:
@rasimx looks like an alternative way to pass this option was implemented in https://github.com/bcoe/c8/pull/423.

It's included in [`c8@7.13.0`](https://github.com/bcoe/c8/pull/428).